clang/include/clang/Basic/Attr.td

```diff
@@ -1183,7 +1183,7 @@ def SYCLKernel : InheritableAttr {
 def SYCLSimd : InheritableAttr {
   let Spellings = [GNU<"sycl_explicit_simd">,
                    CXX11<"intel", "sycl_explicit_simd">];
-  let Subjects = SubjectList<[Function]>;
+  let Subjects = SubjectList<[Function, GlobalVar]>;
   let LangOpts = [SYCLExplicitSIMD];
   let Documentation = [SYCLSimdDocs];
```

Comment:
I would expect to see changes to the documentation for this.

Reply:
Thanks, I will update `AttrDocs.td`. 

I also plan to update the [spec ](https://github.com/intel/llvm/tree/sycl/sycl/doc/extensions/ExplicitSIMD) after finishing the whole SYCL-ESIMD co-existence feature.